views/layout.html

```diff
@@ -54,23 +42,32 @@
         </nav>
       </div>
     {% endif %}
-  </header>
-  <div class="layout">
-    {% block sideNav %}
-      {% if data.page and (not data.piece) and data.page._children.length %}
-        {% set placeheld = false %}
-        <section class="side-nav">
-          {# Side nav: children of the current page #}
-          {% if data.page._children.length %}
-            <nav class="children">
-              {% for page in data.page._children %}
-                <h4><a href="{{ page._url }}">{{ page.title }}</a></h4>
-              {% endfor %}
-            </nav>
-          {% endif %}
-        </section>
+  {% endblock %}
+  {% block pageTitle %}
+    <div class="page-title-wrapper">
+      {% set title = data.page.title %}
+      {% if data.piece %}
+        {% set title = data.piece.title %}
       {% endif %}
-    {% endblock %}
+      <h1 class="page-title">{{ title }}</h1>
```

Comment:
The page title and page subnav should be inside the `main` tag and, typically, an `article` or `section` tag that also wraps the rest of the page content..